### PR TITLE
RSE-1002: Fix: Slowness clicking the job name (migrate job scm status badge and scm action buttons to vue)

### PR DESCRIPTION
**Pull request by L2JE**
Ticket: https://pagerduty.atlassian.net/browse/RSE-1002

QA FW pipe: https://app.circleci.com/pipelines/github/rundeckpro/rundeck-tester/7718/workflows/539e3b77-e6cf-4df5-981a-af39c4198700

Problem:
When having scm enabled and a user clicks on job name in the jobs page the page takes considerably longer than in 4.3.1 because of the controller loading scm data synchronously before rendering the job page.

The scm data is used for the import/export status badges (first image) and the Action Buttons (second image):

image
image

 

Expected Outcome:
Scm plugin data shouldn’t affect the time to render the job page on the browser

 

Proposed Solution:

Load scm data asynchronously by: 
1. Removing Scm processing from `ScheduledExecutionController.show() `endpoint which is accessed when clicking the job name.

2. Load the status badge asynchronously, inserting a vue socket in the `_jobHead.gsp` page and reusing the `JobScmStatus` vue component.

3. For the Action Buttons, add a vue socket to the `_jobActionButtonMenuContent.gsp` page reusing the `JobScmActions` vue component implementation.

### CODE REVIEW

File: rundeckapp/grails-spa/packages/ui-trellis/.eslintrc.js
Comment (DarwisNarvaezDev) [inline rundeckapp/grails-spa/packages/ui-trellis/.eslintrc.js:1]: Why is this file removed?

Comment (L2JE) [reply rundeckapp/grails-spa/packages/ui-trellis/.eslintrc.js:1]: It wasn't mean to be included in the commit, my bad.

**Review by DarwisNarvaezDev — APPROVED** [review]
LGTM
